docs/integrations/sources/snowflake.md

```diff
@@ -0,0 +1,74 @@
+# Snowflake
+
+## Overview
+
+The Snowflake source allows you to sync data from Snowflake.
+It supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+
+This Snowflake source connector is built on top of the source-jdbc code base and is configured to rely on JDBC 3.12.14 [Snowflake driver](https://github.com/snowflakedb/snowflake-jdbc) as described in Snowflake [documentation](https://docs.snowflake.com/en/user-guide/jdbc.html).
+
+#### Resulting schema
+
+The Snowflake source does not alter the schema present in your warehouse. Depending on the destination connected to this source, however, the schema may be altered. See the destination's documentation for more details.
```

Comment:
@irynakruk  here is not clear weather the schema is  altered not. Maybe its good to point exact cases, or rephrase smth.

Reply:
@yaroslav-hrytsaienko thanks, I agree it could be unclear.
I think it would be better to clarify that the result schema could be changed:
  "Depending on the destination connected to this source, however, the result schema may be altered".
Does it look better?